public/static/locales/en/util.json

```diff
@@ -1,4 +1,5 @@
 {
     "details": "Details",
+    "selectPageSize": "Select page size",
```

Comment:
With the addition of this, does this mean the data/apps/analyses pages also have to have `util` added to their `namespacesRequired` array? (Which reminds me, I probably need to do the same for sharing)

Reply:
I don't see specific warning about including `util` when I checked for analyses now but I do see a warning about `app.js` not having `namespacesRequired` array declared. So Let me add that.